lib/ClangImporter/ImportType.cpp

```diff
@@ -75,13 +105,16 @@ namespace {
       /// 'Bool'.
       Boolean,
 
+      /// The source type is 'NSUInteger'.
+      NSUInteger,
```

Comment:
Any reason why you moved this case?

Reply:
All the specific types cases are now together, that's all. (Also, all the nullable cases happen to be together too, but I wasn't specifically optimizing for that.)